lib/client/keystore.go

```diff
@@ -578,8 +582,29 @@ func (fs *fsLocalNonSessionKeyStore) kubeCertPath(idx KeyIndex, kubename string)
 	return keypaths.KubeCertPath(fs.KeyDir, idx.ProxyHost, idx.Username, idx.ClusterName, kubename)
 }
 
+// acquireFileLock is trying to lock the file, until it's successful or timeout is exceeded.
+// File will be created if it doesn't exist.
+func acquireFileLock(filePath string, timeout time.Duration) (func() error, error) {
+	fileLock := flock.New(filePath)
+	ctx, cancel := context.WithTimeout(context.Background(), timeout)
+	defer cancel()
+	if _, err := fileLock.TryLockContext(ctx, 10*time.Millisecond); err != nil {
+		return nil, err
```

Comment:
What msg is in this error? 

In case we couldn't get the lock because Unlock fails or tsh crashes before unlocking, are we showing a good message so the users know to unlock it manually?

Reply:
This one is passed as is, but message is handled up level when we tell user that we couldn't lock known_hosts file.